pkg/controller/core/clusterqueue_controller_test.go

```diff
@@ -229,7 +229,9 @@ func TestUpdateCqStatusIfChanged(t *testing.T) {
 				qManager: qManager,
 			}
 			if tc.newWl != nil {
-				r.qManager.AddOrUpdateWorkload(tc.newWl)
+				if err := r.qManager.AddOrUpdateWorkload(tc.newWl); err != nil {
+					t.Fatalf("Failed to add or update workload : %v", err)
+				}
```

Comment:
```suggestion
				r.qManager.AddOrUpdateWorkload(tc.newWl)
```

Do we need to check it, or can we skip it to stay consistent with the previous logic? Also in other tests.